### ISSUE TITLE
[202405][test_wr_arp] Warm reboot is failing during execution of testWrArpAdvance due to docker exec failure

### ISSUE THREAD

**Issue by gpunathilell** [sonic-net/sonic-buildimage#21236]
#### Description
During `testWrArpAdvance` in sonic-mgmt https://github.com/sonic-net/sonic-mgmt/blob/040d677e9355aa62c7a7ad1e05fc587a22641fa0/tests/arp/test_wr_arp.py#L90 the warm-reboot execution fails due to failed docker exec during the check_docker_exec function during the warm-reboot execution https://github.com/sonic-net/sonic-utilities/blob/2f8508f182566dd75a7c66cfd8dcccaa21470485/scripts/fast-reboot#L469. The docker exec call times out and due to which the warm-reboot execution fails, causing test failure.
Issue is currently observed only in 202405 and with a rare occurence ( 1 in 20 runs)
Log Errors:
```
2024 Dec  2 01:36:01.052167 sonic NOTICE root: Starting warm-reboot
2024 Dec  2 01:36:15.415931 sonic INFO dockerd[1063]: time="2024-12-02T01:36:15.414956087+02:00" level=error msg="Error running exec 5da6854722ff10dd3ce552ff9baba424ac1d6e7a245ca45ebdedc7bbe0521527 in container: exec attach failed: error attaching stdout stream: write unix /var/run/docker.sock->@: write: broken pipe" 
2024 Dec  2 01:36:15.481252 sonic ERR root: Error seen during warm-reboot shutdown process: Docker exec on lldp timedout
2024 Dec  2 01:36:22.245221 sonic NOTICE root: Cancel warm-reboot: code (0)
```

Only reproduced once in 20 runs, and the docker for which the docker exec fails is different in each case

#### Steps to reproduce the issue:
1. Run testWrArpAdvance test

**Comment by bingwang-ms** [comment]
@vaibhavhd @saiarcot895 Can you help take a look at this issue?

**Comment by vaibhavhd** [comment]
Yes, @gpunathilell , can you please add logs for the failure run? Syslog of the shutdown path (where the docker exec timed out) will help here.

Docker exec can timeout or can take longer when the CPU is too busy or when the container (where exec is issued) is not responding.
It could be dockerd related too. We need logs for triaging this failure.

**Comment by gpunathilell** [comment]
The issue is seen only on a single device, One reproduction is seen after the original issue was detected (after 20 tries). Will share the logs

**Comment by gpunathilell** [comment]
@vaibhavhd Could you let me know what steps you mentioned in order to check the system where the issue occurred?

**Comment by gpunathilell** [comment]
@vaibhavhd Please check the analysis which I have added in the description of the issue

**Comment by gpunathilell** [comment]
@vaibhavhd There's a PR for fixing by re-trying this docker exec as issue seems to be due to utilization. Please review: https://github.com/sonic-net/sonic-utilities/pull/3772

**Comment by gpunathilell** [comment]
Hi @vaibhavhd, New issue created: https://github.com/sonic-net/sonic-buildimage/issues/21920. Please refer that one and add comments if needed. Thank you
